benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -10878,9 +10876,8 @@ exports[`Render Summary for multi-table.table.html 1`] = `
        >
         <div
          className="cell"
-        >
-         -
```

Comment:
These are status cells that are in `local summary`. I believe they should also be `0` as the rows are being counted, right

Reply:
@PhilippWendler  :point_up:

Reply:
No, `0` would actually be wrong here, because the local summary is shown only if there are such statistics, and of course these do not exist for only an empty set of rows. The number here should actually be always equal to the total number of results, but we don't show it because it would be redundant.

Reply:
Well, we definitely have rows in the local summary in this case. Do they maybe not have any status in the local summary? 
Here is a screenshot for context (the `0` in status comes from my current changes) 
![Screenshot from 2020-12-11 09-11-56](https://user-images.githubusercontent.com/24787761/101879107-15fb0080-3b91-11eb-899a-91898f7aa24a.png)

Reply:
Yes, i took a step back and read through the code again. 
The issue as missing differentiation between `undefined` and `null`. 
`undefined` represents a missing result caused by no available data, and usually gets represented by a `0` for status values. 
`null` gets passed on by the stats table and makes the code default to `-`

Reply:
Ok, but I would suggest to not use `undefined` *and* `null`, this gets highly confusing. The code that counts the rows per category for the status column should simply return 0.

Reply:
Yeah, this is caused by the way that the stats are prepared. The categorization is done on the fly and buckets (like `correct-true` and so on) will only be created if we have lines matching this criteria. This is why some cells in the stats table will receive a `undefined` value if there are no matching lines for this category. 
Actually, `local summary` is the only edge-case where we need to differentiate between these two values as this is the only place where we retrieve values directly from the `stats` object of the data-set.